python/mvdtool.cpp

```diff
@@ -167,6 +198,11 @@ PYBIND11_MODULE(mvdtool, mvd) {
                 auto res = f.getPositions(r);
                 return py::array({res.shape()[0], res.shape()[1]}, res.data());
              })
+        .def("positions", [](const File& f, const pyarray<size_t>& idx) {
+                const auto& func = [&f](const MVD::Range& r){return f.getPositions(r);};                
+                const auto res = _atIndicesMulti<Positions, 3>(func, f.size(), idx);
+                return py::array({res.size(), res[0].size()}, res[0].data());
```

Comment:
Not sure why the `res[0].data` is necessary here but otherwise I would get an higher level double-addressable structure.

Reply:
res is a `vector<array<T>>` so indeed a good way of getting the base address is doing `data()` in the first array of the vector.